release/Dockerfile.release

```diff
@@ -45,7 +52,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 RUN easy_install pip
 RUN pip install --upgrade six pyyaml google-api-python-client \
```

Comment:
suggest using `Pipfile` to lock down python dependencies

Reply:
I added a TODO. I tried to make it work but ran into problems. I'd rather get this PR submitted because it fixes our releases which is needed to get a Docker image out that works for the FE.
So I don't think its worth delaying to try to make pipenv work.